custom/icds_reports/urls.py

```diff
@@ -151,6 +151,6 @@
     url(r'^inactive_aww', InactiveAWW.as_view(), name='inactive_aww'),
     url(r'^inactive_dashboard_users', InactiveDashboardUsers.as_view(), name='inactive_aww'),
     url(r'^health_indicators', DishaAPIView.as_view(), name='disha_api'),
-    url(r'^new_health_indicators', DishaAPIViewV2.as_view(), name='disha_api_v2'),
+    url(r'^nic_health_indicators', DishaAPIViewV2.as_view(), name='nic_disha_api'),
```

Comment:
should probably change the url name and class name here as well if it is not related to DISHA